onfido-java/src/main/java/com/onfido/Onfido.java

```diff
@@ -133,12 +135,26 @@ public Builder apiToken(String apiToken) {
 
     /**
      * Interceptor attribute.
+     * @deprecated
+     * Use {@link Builder#clientInterceptors(List)} instead.
      *
      * @param interceptor the HTTP interceptor
      * @return the builder
      */
-    public Builder clientInterceptor(Interceptor interceptor) {
-      this.clientInterceptor = interceptor;
+    @Deprecated
```

Comment:
As a new version of library has not been delivered yet after this method has added, we don't need to deprecate it before removing. Could you please just remove the method?

Reply:
@dvacca I'm afraid, it's not a good idea, since I'm already using this method in the older version of the library 2.6.0. So the API is already in production used by the projects and we'd break the backwards compatibility without the warning.

Reply:
All right, good spot. I though that was part of your previous [PR#79](https://github.com/onfido/onfido-java/pull/79/files) but that was not the case so good to deprecate it first. 👍